app/scripts/lib/setupSentry.js

```diff
@@ -473,7 +58,14 @@ export default function setupSentry() {
 function getClientOptions() {
   const environment = getSentryEnvironment();
   const sentryTarget = getSentryTarget();
-
+  browser.management
```

Comment:
Nit: Usually when we call an async function without an `await`, it's a mistake. I know you're doing it on purpose in this case, but a comment would be helpful in explaining why, so that readers don't think we accidentally forgot it.